tests/utils/verification/volumeproperties.go

```diff
@@ -143,24 +143,34 @@ func getVolumeStatusHost(name, hostName string) string {
 func VerifyDetachedStatus(name, hostName, esxName string) bool {
```

Comment:
Let the new functionality be the preferred way and lets replace whats there without having to change test code.